common/index.html

```diff
@@ -20,6 +20,30 @@
     <meta name="twitter:creator" content="<%= htmlWebpackPlugin.options.twitter.creator %>" />
     <meta name="google-site-verification" content="dRWkvANAUNAhNyMnTyc7M7S3lnucotMY8j8R-gsZhbo" />
     <link rel="manifest" href="/manifest.json" />
+
+    <script> (function (d, s) {
+      var js = d.createElement(s),
+        sc = d.getElementsByTagName(s)[0];
+      js.src = "https://paywall.unlock-protocol.com/static/unlock.1.0.min.js";
+      sc.parentNode.insertBefore(js, sc);
+    }(document, "script"));
+    </script>
+
+    <script>
+      var unlockProtocolConfig = {
+        "icon": "https://assets.unlock-protocol.com/tmp/my-crypto.png",
```

Comment:
Can we make this a local loaded asset instead of linking to `assets.unlock-protocol.com`

Reply:
Yes you can! I can update the PR for this.